apps/web/components/apps/AppPage.tsx

```diff
@@ -89,6 +89,8 @@ export const AppPage = ({
 
   const [existingCredentials, setExistingCredentials] = useState<number[]>([]);
   const [showDisconnectIntegration, setShowDisconnectIntegration] = useState(false);
+  // FIXME: remove hardcoding
+  const showSetupIntegration = slug === "alby";
```

Comment:
We can omit this for now, since you have the "This app has not been setup yet" and action button there I think it works great now.